backend/product/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from core.models import MainModel
+
```

Comment:
Another model that we will also need is `VehicleType` with `name` attribute as well, which the `Vehicle` model will have a foreign key to